src/sentry/static/sentry/app/components/releaseStats.jsx

```diff
@@ -22,16 +22,18 @@ const ReleaseStats = React.createClass({
       return null;
     }
     return (
-      <div className="release-info">
-        <div><b>{commitCount}{t(' commits by ')}{authorCount}{t(' authors')}</b></div>
-        {release.authors.map(author => {
-          return (
-            <span className="assignee-selector tip"
-                 title={author.name + ' ' + author.email}>
-              <Avatar user={author}/>
-            </span>
-          );
-        })}
+      <div className="release-stats">
+        <h6>{commitCount}{t(' commits by ')}{authorCount}{t(' authors')}</h6>
```

Comment:
I'll fix, but this is something that already existed (and presumably already in transifex). I only changed the tags that wrap it. Do we need to purge those or something?